test/views/NotificationsViewSpec.scala

```diff
@@ -56,6 +60,21 @@ class NotificationsViewSpec extends UnitViewSpec with Stubs {
         view mustNot containElementWithID("submission_notifications-table-row0-amend_link")
       }
 
+      "status Accepted during BST" in {
+        val view = createView(
+          submission,
```

Comment:
👍 - might be clearer if the "before BST" test explicitly defined the test date/time rather than rely on the test default.

Reply:
I have addressed your comment.